sdk/identity/azure-identity/samples/azure-aad-auth-with-redis-py.md

```diff
@@ -0,0 +1,143 @@
+## Azure Cache for Redis: Azure AD with redis client library
+
+### Table of contents
+
+- [Prerequisites](#prerequisites)
+- [Samples guidance](#samples-guidance)
+- [Authenticate with Azure AD - Hello World](#authenticate-with-azure-ad-hello-world)
+- [Authenticate with Azure AD - Handle Re-authentication](#authenticate-with-azure-ad-handle-re-authentication)
+- [Troubleshooting](#troubleshooting)
+
+#### Prerequisites
+
+- Configuration of Role and Role Assignments is required before using the sample code in this document.
```

Comment:
I left this same feedback for Vinay in his PR. We should link to a doc that explains how to configure roles and role assignments. Since it's a prereq, we don't want customers to be blocked if they can't get past that.

Reply:
@scottaddie Talked to the feature team. The doc will not be ready before our release which is on 8/9 and they would like us to merge the PR w/o the link and add the link when it is ready.

Do you have concerns?

Reply:
No concerns. We can update later.